app/src/main/java/org/pesmypetcare/mypetcare/features/pets/PetHealthInfo.java

```diff
@@ -0,0 +1,426 @@
+package org.pesmypetcare.mypetcare.features.pets;
+
+import org.pesmypetcare.usermanager.datacontainers.DateTime;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+public class PetHealthInfo {
+    private Map<DateTime, Double> weight;
+    private Map<DateTime, Double> dailyKiloCalories;
+    private Map<DateTime, Integer> exerciseFrequency;
+    private Map<DateTime, Integer> weeklyExercise;
+    private Map<DateTime, Double> weeklyKiloCaloriesAverage;
+    private Map<DateTime, Integer> washFrequency;
+    private Map<DateTime, Integer> weeklyMeals;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'weeklyMeals' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5421176)